bin/cimpler

```diff
@@ -30,6 +34,11 @@ args           = require('optimist')
 
 var command = args._[0];
 
+connOptions = {
```

Comment:
I was actually pointing out that the `var` is missing so it's a global variable. I do like `connectionOptions` more than `connOptions`, though!